app/vmagent/remotewrite/remotewrite.go

```diff
@@ -103,7 +103,7 @@ var (
 	streamAggrDropInputLabels = flagutil.NewArrayString("streamAggr.dropInputLabels", "An optional list of labels to drop from samples "+
 		"before stream de-duplication and aggregation . See https://docs.victoriametrics.com/stream-aggregation.html#dropping-unneeded-labels")
 
-	disableOnDiskQueue = flag.Bool("remoteWrite.disableOnDiskQueue", false, "Whether to disable storing pending data to -remoteWrite.tmpDataPath "+
+	disableOnDiskQueuePerURL = flagutil.NewArrayBool("remoteWrite.disableOnDiskQueue", "Whether to disable storing pending data to -remoteWrite.tmpDataPath "+
```

Comment:
Looks like this flag is related to `remoteWrite.dropSamplesOnOverload`. I think we should make `remoteWrite.dropSamplesOnOverload` to be configurable per-url as well. wdyt?

Reply:
Sure thing :)